client.go

```diff
@@ -408,6 +409,9 @@ func (c *Client) url(elem ...string) string {
 func unixDial(networ, addr string) (net.Conn, error) {
 	var raddr *net.UnixAddr
 	var err error
+	var result net.Conn
+
+	shared.Debugf("In unixDial, with addr: %s", addr)
```

Comment:
Was that just temporary debug or do you think this is actually useful to keep around?